optuna_dashboard/_serializer.py

```diff
@@ -166,6 +167,7 @@ def serialize_study_detail(
     if serialized["is_preferential"]:
         serialized["preference_history"] = serialize_preference_history(system_attrs)
         serialized["preferences"] = get_preferences(system_attrs)
+        serialized["skipped_trials"] = skipped_trials
```

Comment:
A nit, but could you update the attribute name?

```suggestion
        serialized["skipped_trial_numbers"] = skipped_trials
```